pkg/helmreconciler/pruning.go

```diff
@@ -24,22 +24,22 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func (h *HelmReconciler) prune(all bool) error {
+func (h *HelmReconciler) Prune(all bool) error {
```

Comment:
Can we add a comment for Prune?  Is it to remove dups?

Reply:
Done.